pkg/k8s/slim/k8s/apis/core/v1/types.go

```diff
@@ -474,10 +486,6 @@ type ServiceSpec struct {
 	// and ExternalTrafficPolicy is set to Local.
 	// +optional
 	HealthCheckNodePort int32 `json:"healthCheckNodePort,omitempty" protobuf:"bytes,12,opt,name=healthCheckNodePort"`
-
-	// sessionAffinityConfig contains the configurations of session affinity.
-	// +optional
-	SessionAffinityConfig *SessionAffinityConfig `json:"sessionAffinityConfig,omitempty" protobuf:"bytes,14,opt,name=sessionAffinityConfig"`
```

Comment:
Likewise, is this move meaningful?

Reply:
I thought that `SessionAffinityConfig` should follow `SessionAffinity` field in the upstream types.go. But it's not the case - https://raw.githubusercontent.com/kubernetes/kubernetes/v1.19.0-rc.2/staging/src/k8s.io/api/core/v1/types.go. Let me undo the ordering.